docs/using/examples/environment.rst

```diff
@@ -16,7 +16,7 @@ Create the Virtual Machines
 You can reuse the Virtual Machines defined in the Vagrant configuration for :ref:`the MongoDB tutorial <tutmongo>`.
 If you have since shutdown or destroyed those VMs, boot them up again:
 
-.. code-block:: console
+.. prompt:: bash alice@mercury:~/flocker-tutorial$ auto
 
    alice@mercury:~/flocker-tutorial$ vagrant up
    Bringing machine 'node1' up with 'virtualbox' provider...
```

Comment:
I find this a bit confusing, but it could be because I don't know what 'auto' is supposed to do here.

we define the prompt as:
```
bash alice@mercury:~/flocker-tutorial$ auto
```

but then list the steps as:

```
 alice@mercury:~/flocker-tutorial$ vagrant up
```
my first thought without looking up the HTML that resulted from it, was that this would result as:
```
alice@mercury:~/flocker-tutorial$  alice@mercury:~/flocker-tutorial$ vagrant up
```